src/styles/_generalSettings.scss

```diff
@@ -0,0 +1,38 @@
+@import './variables';
```

Comment:
Tabulation in this file looks weird and large. Doesn't our prettier configured to catch this big tabulation in scss files. If no, could you please add this configuration